plugin.go

```diff
@@ -37,9 +40,6 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	var sync bool
 	var dryRun bool
 	var override string
-	var username string
-	var password string
-	var vaultpass string
```

Comment:
Even though user password and vaultpass will be stored in keyring, we still need the ability to pass them to command as option (along with `--keyring-passphrase`) to add them to keyring